Example/SlideController/MainController.swift

```diff
@@ -11,21 +11,21 @@ import SlideController
 
 class MainController {
     fileprivate let _view = MainView()
-    fileprivate let _scrollControler : ScrollController<MainTitleScrollView, MainTitleItem>!
+    fileprivate let _scrollControler : SlideController<MainTitleScrollView, MainTitleItem>!
```

Comment:
`private let scrollControler: SlideController<MainTitleScrollView, MainTitleItem>!`
As I know private can be used in Extensions in same file if you use Swift4